tools/packages.yaml

```diff
@@ -7266,6 +7266,11 @@ url: https://x.org/archive/individual/app
 activity: none
 ---
 kind: url
+name: repo
+url: https://gerrit.googlesource.com/git-repo
```

Comment:
@uberhacker does this URL work for your update check script?

Does `https://gerrit.googlesource.com/git-repo/+refs` ?

(Working with his update check script is the important for the urls we have in packages.yaml.)